pkg/apis/cincinnati/v1alpha1/cincinnati_types.go

```diff
@@ -39,6 +39,12 @@ type CincinnatiSpec struct {
 	// specified, it takes precedence over GitHubOrg and GitHubRepo and disables
 	// retrieval of graph data from GitHub.
 	GraphDataImage string `json:"graphDataImage,omitempty"`
+
+	// CertConfigMapKey refers to the ConfigMap key that holds a CA cert to the registry
+	// Cincinnati will contact to build the upgrade graph.  The name of the
+	// ConfigMap holding the key comes from the field additionalTrustedCA
+	// in the image.config.openshift.io API.
+	CertConfigMapKey string `json:"certConfigMapKey"`
```

Comment:
I think this should be optional, particularly for connected use cases.

Reply:
Right, it's optional.  If you set a key and your registry doesn't take one you'll have errors, so `""`, or not including it at all, is a perfectly acceptable option.

Reply:
Ah, I was thinking it should have `omitempty`, but perhaps defaulting to empty string is enough.

Reply:
My thinking here is that we don't have API overload and it's good to raise awareness of this key. Could go either way here.

Reply:
My vote is for `omitempty`

Reply:
```suggestion
	// +kubebuilder:validation:Optional
	CertConfigMapKey string `json:"certConfigMapKey"`
```

Reply:
we'll `omitempty` then.